lib/firefox.js

```diff
@@ -20,6 +20,7 @@ module.exports = async function (source) {
     'sign',
     '--source-dir', source,
     '--api-key', process.env.WEXT_SHIPIT_FIREFOX_JWT_ISSUER,
+    '--ignore-files', '.api-key package.json package-lock.json yarn.lock .npmrc .yarnc',
```

Comment:
I don't think that `.api-key` is a standardized file name 🤔 what's the rationale for including it here?

Reply:
Hi,
It's an example from Mozilla MDN reference documentation https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference#Global_options

(Looks like some people do store their keys in a file of format  "filename.api-key" )

However, I tried my changes from my gitlab and it didn't consider ignore-files option.. I raised an issue regarding that - it didn't seem to work when you call it within nodejs script.  https://github.com/mozilla/web-ext/issues/1333

In an earlier issue there https://github.com/mozilla/web-ext/issues/843 , they did add ignore-files support for 'sign' - but that doesn't seem to pick for when you call it programmatically.. I guess "web-ext sign" from a CLI might work with ignore-files after that fix which they closed.